log/slow/parser.go

```diff
@@ -358,7 +358,11 @@ func (p *SlowLogParser) sendEvent(inHeader bool, inQuery bool) {
 		// Started parsing in header after Query_time.  Throw away event.
 		return
 	}
-
+	// If the query was logged because it took more than RateMaxQueryTime
+	// the rate should be ignored to prevent a wrong sampling
```

Comment:
I think this comment leaves open question too like "What do you mean by 'wrong sampling'?" and "why ignore 'the rate'?" (do you mean ignore the query?), etc. In other words: make it more clear what we're doing here and why, how it relates to `slow_query_log_always_write_time` (consequently, this only affects Percona Server), etc.